panels/spoolman.py

```diff
@@ -0,0 +1,405 @@
+                'serve" xmlns="http://www.w3.org/2000/svg"><path style="stroke:#6e0b30;stroke-width:0;stroke-dasharray:'
+                'none;stroke-linecap:butt;stroke-dashoffset:0;stroke-linejoin:miter;stroke-miterlimit:4;fill:#343434;fi'
+                'll-rule:nonzero;opacity:1" vector-effect="non-scaling-stroke" transform="matrix(.58757 0 0 3.94769 197'
+                '.135 250.047)" d="M0-63.27c34.925 0 63.27 28.345 63.27 63.27 0 34.925-28.345 63.27-63.27 63.27-34.925 '
+                '0-63.27-28.345-63.27-63.27 0-34.925 28.345-63.27 63.27-63.27z"/><path style="stroke:#6e0b30;stroke-wid'
+                'th:0;stroke-dasharray:none;stroke-linecap:butt;stroke-dashoffset:0;stroke-linejoin:miter;stroke-miterl'
+                f'imit:4;fill:#{self.filament.color_hex};fill-rule:nonzero;opacity:1" vector-effect="non-scaling-stroke'
+                '" transform="matrix(.38158 0 0 3.46232 197.135 250.047)" d="M0-63.27c34.925 0 63.27 28.345 63.27 63.27'
+                ' 0 34.925-28.345 63.27-63.27 63.27-34.925 0-63.27-28.345-63.27-63.27 0-34.925 28.345-63.27 63.27-63.27'
+                'z"/><path style="stroke:#000;stroke-width:0;stroke-dasharray:none;stroke-linecap:butt;stroke-dashoffse'
+                f't:0;stroke-linejoin:miter;stroke-miterlimit:4;fill:#{self.filament.color_hex};fill-rule:nonzero;opaci'
+                'ty:1" vector-effect="non-scaling-stroke" transform="matrix(2.07364 0 0 3.3577 117.295 250.047)" d="M-3'
+                '8.503-65.24h77.006V65.24h-77.006z"/><g transform="matrix(.58757 0 0 3.94769 37.454 250.047)"><filter i'
+                'd="a" y="-.057" height="1.114" x="-.057" width="1.272"><feGaussianBlur in="SourceAlpha" stdDeviation="'
+                '3"/><feOffset dx="20" result="oBlur"/><feFlood flood-color="rgb(0,0,0)" flood-opacity=".67"/><feCompos'
+                'ite in2="oBlur" operator="in"/><feMerge><feMergeNode/><feMergeNode in="SourceGraphic"/></feMerge></fil'
+                'ter><path style="stroke:#6e0b30;stroke-width:0;stroke-dasharray:none;stroke-linecap:butt;stroke-dashof'
+                'fset:0;stroke-linejoin:miter;stroke-miterlimit:4;fill:#343434;fill-rule:nonzero;opacity:1;filter:url(#'
+                'a)" vector-effect="non-scaling-stroke" d="M0-63.27c34.925 0 63.27 28.345 63.27 63.27 0 34.925-28.345 6'
+                '3.27-63.27 63.27-34.925 0-63.27-28.345-63.27-63.27 0-34.925 28.345-63.27 63.27-63.27z"/><path style="s'
+                'troke:#6e0b30;stroke-width:0;stroke-dasharray:none;stroke-linecap:butt;stroke-dashoffset:0;stroke-line'
+                'join:miter;stroke-miterlimit:4;fill:#111;fill-rule:nonzero;opacity:1" vector-effect="non-scaling-strok'
+                'e" transform="scale(.41452)" d="M0-63.27c34.925 0 63.27 28.345 63.27 63.27 0 34.925-28.345 63.27-63.27'
+                ' 63.27-34.925 0-63.27-28.345-63.27-63.27 0-34.925 28.345-63.27 63.27-63.27z"/></g></svg>').encode())
```

Comment:
this may OK for an initial merge, but long term this will be changed, since it is not theme friendly